packages/lib/plugin-connector/src/data-sources/get-sources.spec.js

```diff
@@ -87,4 +87,10 @@ describe('loadSourceOptions', () => {
 		expect('ssl' in result).toBeTruthy();
 		expect('rejectUnauthorized' in result.ssl).toBeTruthy();
 	});
+	it('should pickup boolean flags', () => {
+		vi.stubEnv('EVIDENCE_SOURCE__source__ssl__rejectUnauthorized', 'false');
+		const result = loadSourceOptions('source');
+		expect('ssl' in result).toBeTruthy();
+		expect('rejectUnauthorized' in result.ssl).toBeFalsy(); // <-- this is failing
```

Comment:
```suggestion
		expect('rejectUnauthorized' in result.ssl).toBeTruthy();
		expect(result.ssl.rejectUnauthorized).toBeFalsey();
```

Reply:
@ItsMeBrianD 
`expect(result.ssl.rejectUnauthorized).toBeFalsey();` will also fail as `rejectUnauthorized` is being parsed as string here. `"false"` is a truthy value, as @ud3sh mentioned.

Reply:
Correct, but this test _should_ be passing

Reply:
I meant to say, in the suggested changes
line 94 - `expect('rejectUnauthorized' in result.ssl).toBeTruthy();` - this will pass
line 95 - `expect(result.ssl.rejectUnauthorized).toBeFalsey();` - this will not pass